src/drivers/intel/cavs/alh.c

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#include <stdint.h>
```

Comment:
The rule is the most global headers at the end and also in alphabetical order across their own group:
#include <sof/alh.h>
#include <sof/dai.h>
#include <sof/dma.h>
#include <sof/memory.h>
#include <sof/trace.h>
#include <ipc/dai.h>
#include <stdint.h>